x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/machine_learning/machine_learning_setup.ts

```diff
@@ -22,7 +23,7 @@ export const executeSetupModuleRequest = async ({
     .set(getCommonRequestHeader('1'))
     .send({
       prefix: '',
-      groups: ['auditbeat'],
+      groups: [ML_GROUP_ID],
```

Comment:
Amazing job sticking with it to figure this out. Woohoo to finally having our ML tests able to run.